### ISSUE TITLE
Test for WCAG technique H67: Using null alt text and no title attribute on img elements for images that AT should ignore

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#206]
http://www.w3.org/TR/WCAG20-TECHS/H67.html
The purpose of this technique is to show how images can be marked so that they can be ignored by Assistive Technology.

If no title attribute is used, and the alt text is set to null (i.e. alt="") it indicates to assistive technology that the image can be safely ignored.

Note: Have a "null" alt attribute is not the same as having no alt attribute. 
## Test procedure

For each image that should be ignored:
- [ ] Check that title attribute is either absent or empty.
- [ ] Check that alt attribute is present and empty or contains only whitespace (but not nbsp) 
### Scope

Image
### Applicable

Image that should be ignored
### Not applicable

Image that conveys meaning
### Related

F38 is counterpart of this test
### Remarks

Is it testable that an image should be ignored or should we just test if this technique is used? As this is a technique, we should just check whether or not alt="" is used. If it's used, this is a pass, if this technique isn't used, it's a fail.

if we want to differentiate between n/a and fail, we could build a checker if an image should have an empty or non empty alt: 
- an image in a data table should have an description as that is normally used for a category (checked, unchecked etc)
- an image in a list (definition list, ordered list) should have a descriptive tect
- Images outside the content area could be ignored, as that is always decorative (logo and identity).
